### DISCUSSION TITLE
Formatting a flash drive?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [BlitterStudio/amiberry#1098] [Q&A] [answered]
Hi

I was puttting a flash drive in USB port and it could not see it. Any suggestions? How can I format it and what file systems does it support?

Can it support usb hard drives?

Thanks
Ward

**Comment by midwan** [comment, marked as answer]
USB drives will be seen as any other drive available in the system, through Linux.
If you want to format it under AmigaOS, you will also need root privileges on the drive, through Linux, in order to do that.
Then of course you will need to mount it as an HDD (by selecting the relevant option in the GUI, and providing the full device path), after which it will be made available in AmigaOS.

The rest is the normal procedure from inside AmigaOS itself (e.g. HDToolbox to partition it, then format it as usual).
